CHANGELOG.rst

```diff
@@ -1,6 +1,109 @@
+
+  - [R] The ``mlflow_run`` API for running MLflow projects has been modified to more closely reflect the Python ``mlflow.run`` API. In particular, the order of the ``uri`` and ``entry_point`` arguments has been reversed and the ``param_list`` argument has been renamed to ``parameters``. (#1265, @smurching)
+  - [R] The ``mlflow_cli`` and ``crate`` APIs are now private. Also, the return values of experiment CRUD APIs have been updated to more closely match the REST API. In particular, ``mlflow_create_experiment`` now returns a string experiment ID instead of an experiment, and the other APIs return NULL. (#1246, @smurching)
+  - [R] Remove ``mlflow_snapshot`` and ``mlflow_restore_snapshot`` APIs. New R APIs will be introduced after MLflow 1.0 for managing code and model dependencies. Also, the ``r_dependencies`` argument used to specify the path to a packrat r-dependencies.txt file has been removed from all APIs. (#1263, @smurching)
+
+- [Artifacts] In APIs outside of Tracking, an artifact's location is now represented as a URI. (#1190, #1254, @dbczumar; #1174, @dbczumar, @sueann; #1206, @tomasatdatabricks)
+
+  - The affected APIs are:
+
+    - Python: ``<model-type>.load_model``, ``azureml.build_image``, ``sagemaker.deploy``, ``sagemaker.run_local``, ``pyfunc._load_model_env``, ``pyfunc.load_pyfunc``, and ``pyfunc.spark_udf``
+    - R: ``mlflow_load_model``, ``mlflow_rfunc_predict``, ``mlflow_rfunc_serve``
+    - CLI: ``mlflow models serve``, ``mlflow models predict``
+  - To allow referring to artifacts in the context of a run, we introduce a new URI scheme of the form ``runs:/<run_id>/relative/path/to/artifact``. (#1169, #1175, @sueann)
+
+- [Runs] The ``user`` property of Runs has been moved to tags (similarly, the ``run_name``, ``source_type``, ``source_name`` properties were moved to tags in 0.9.0). (#1230, @acroz; #1275, #1276, @aarondav)
+- [Runs][Python] Expose ``RunData`` fields (``metrics``, ``params``, ``tags``) as dictionaries (#1078, @smurching)
+- [Runs] ``RunInfo.status``'s type is now string (#1264, @mparkhe)
+- [Installation] The MLflow python package no longer depends on ``scikit-learn``, ``mleap``, or ``boto3``. Users who want to use the ``scikit-learn`` support, the ``MLeap`` support, or ``s3`` artifact repository / ``sagemaker`` support will have to install these respective dependencies explicitly. (#1223, @aarondav)
+
+More features and improvements:
+
+- [Tracking][DB] Non-default driver support for SQLAlchemy backends: ``db+driver`` is now a valid tracking backend URI scheme (#1297, @drewmcdonald)
+- [Tracking] Validate backend store URI before starting tracking server (#1218, @luke-zhu, @sueann)
+- Add ``view_type`` argument to ``MlflowClient.list_experiments()`` in Python. (#1212, @smurching)
+- [Python] Dictionary values provided to ``mlflow.log_params`` and ``mlflow.set_tags`` can now be non-string types (e.g., numbers), and they will be automatically converted to strings. (#1364, @aarondav)
```

Comment:
they will be automatically converted to strings. -> they are automatically converted to strings.